scipy/signal/filter_design.py

```diff
@@ -3653,6 +3653,232 @@ def besselap(N, norm='phase'):
     return asarray([]), asarray(p, dtype=complex), float(k)
 
 
+def iirnotch(w0, bw, rp=3):
+    """
+    Design second-order IIR notch digital filter.
+
+    A notch filter is a band-stop filter with a narrow bandwidth
+    (high quality factor). It rejects a narrow frequency band and
+    leaves the rest of the spectrum little changed.
+
+    Parameters
+    ----------
+    w0 : float
+        Normalized frequency to remove from a signal. It is a
+        scalar that must satisfy  0 < w0 < 1, with w0 = 1 corresponding
+        to half of the sampling frequency.
+    bw : float
+        Normalized filter bandwidth (the diference between up and lower
+        passband edges).
+    rp : float, optional
```

Comment:
I think it should be called `rs`. According to http://scipy.github.io/devdocs/generated/scipy.signal.iirfilter.html#scipy.signal.iirfilter: "For Chebyshev and elliptic filters, provides **the minimum attenuation in the stop band.** (dB)".

By the way let's use the same definition: "The minimum attenuation in the stop band (determined by `bw`) expressed in dB. Default is 3." Does it sound good?